acme/acme/standalone.py

```diff
@@ -203,9 +236,17 @@ class HTTP01RequestHandler(BaseHTTPServer.BaseHTTPRequestHandler):
 
     def __init__(self, *args, **kwargs):
         self.simple_http_resources = kwargs.pop("simple_http_resources", set())
-        self.timeout = kwargs.pop('timeout', 30)
+        self._timeout = kwargs.pop('timeout', 30)
         BaseHTTPServer.BaseHTTPRequestHandler.__init__(self, *args, **kwargs)
 
+    @property
+    def timeout(self):
```

Comment:
Done. To be a little more French here and complain, I just do not get why Python developers made things like this. I guess there is a decent reason to explain why handlers are passed by class references or factory methods to `HTTPServer` objects, certainly something related to thread safety. But why defining handler customizations through class-level properties, instead of having a dedicated object/dict that you would pass to `HTTPServer` constructors to define things like timeouts? It would have been way cleaner to use...